doc-content/enterprise-only/employee-rostering/er-tenant-create-proc.adoc

```diff
@@ -7,9 +7,7 @@ IMPORTANT: You cannot change tenant parameters after you create the tenant.
 
 .Procedure
 
-.  To create a new tenant in the Optaweb employee rostering application web interface, in the top right part of the browser window click the *Admin* tab.
-
-. Click *Add Tenant*.
+.  To create a new tenant in the OptaWeb employee rostering application web interface, in the top right corner of the browser window click the Settings (gear) icon then click *Add*.
```

Comment:
Question - does the "Settings" title appear when you hover over the gear icon?

Suggestion - if *Add* appears as a drop-down element, maybe changing "click" to "select" and "then" to "and".

/the Settings (gear) icon then click *Add*./the settings gear icon and select *Add*./the *Settings* gear icon and select *Add*.

Reply:
To answer the first question, no, the "Settings" title does not appear when you hover over the gear icon.